reflex/.templates/apps/blank/code/blank.py

```diff
@@ -16,19 +16,33 @@ def index() -> rx.Component:
     return rx.container(
         rx.color_mode.button(position="top-right"),
         rx.vstack(
-            rx.heading("Welcome to Reflex!", size="9"),
+            rx.heading("Welcome to Reflex 🦕", size="9"),
+            rx.text("Need help? 🤔 Check out the documentation."),
             rx.text(
-                "Get started by editing ",
-                rx.code(f"{config.app_name}/{config.app_name}.py"),
-                size="5",
+                "You can edit this page by modifying the code in the file below. Happy coding 🎉",
             ),
-            rx.link(
-                rx.button("Check out our docs!"),
-                href="https://reflex.dev/docs/getting-started/introduction/",
-                is_external=True,
+            rx.code(f"{config.app_name}/{config.app_name}.py"),
+            rx.flex(
+                rx.card(
+                    rx.link(
+                        rx.flex(
+                            rx.avatar(src="https://reflex.dev/reflex_banner.png"),
+                            rx.box(
+                                rx.heading("Quick Start 🚀"),
+                                rx.text("Get started with Reflex before your coffee ☕"),
+                            ),
+                            spacing="3",
+                        ),
+                        href="https://reflex.dev/docs/getting-started/introduction/",
+                        target="_blank",
+                        underline="none"
+                    ),
+                    as_child=True,
+                )
             ),
             spacing="5",
             justify="center",
+            align="center",
```

Comment:
personal taste, i don't really like the centered lines of text

i'm okay with centering the button though